apis/pkg/meta/configuration_types.go

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
Note that the newly introduced `pkg/meta` and `pkg/meta/v1` types are identical to the existing `pkg/meta/v1alpha1` types.